### PR TITLE
fix: Ticket dashboard  table css overflow on small screens

### LINKED ISSUE #3589: Ticket Dashboard: Table CSS Overflow on Small Device Screens
**Describe the bug**

Table CSS Overflow on Small Device Screens.

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to 'Event/{event.id}/tickets'
2. See the screen with device `width < 690px`
3. See error

**Expected behaviour**

Fix the CSS Overflow

**Screenshots**

Screenshot 2019-11-03 at 5 38 13 PM

**Smartphone (please complete the following information):**
 - Device: iPhone 8 & old
 - Browser: All

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #3589 #3605 

#### Short description of what this resolves:
fixes the ticket dashboard Table CSS Overflow on Small Device Screens.
issuebug

Also fixes the manage events dashboard search-box overflow on small screen devices :
Screenshot from 2019-11-05 20-11-08

#### Changes proposed in this pull request:
I have fixed it by setting the css overflow property to scroll.

Screenshot from 2019-11-03 22-07-53

Screenshot from 2019-11-03 22-07-59

Screenshot from 2019-11-07 21-23-37

#### Checklist

- [x]  I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
Can you provide a GIF for the same, Its difficult to understand this change from the SS ?

**Comment by snitin315** [comment]
@kushthedude  here is the gif :
overflow

**Comment by kushthedude** [comment]
> @kushthedude here is the gif :
> overflow

Use properties ` x-scrollable`, defined in utils.

**Comment by snitin315** [comment]
> Use properties ` x-scrollable`, defined in utils.
@kushthedude  Done . please review

File: app/templates/events/view/tickets/index.hbs [hunk 1]
```diff
@@ -10,13 +10,14 @@
...
 <br>
-<div class="row">
+<div class="row x-scrollable">
   <div class="ui header">
     {{t 'Order Summary'}}
     <span class="sub header">
       {{t 'By order status'}}
     </span>
   </div>
+
```
Comment (kushthedude) [inline app/templates/events/view/tickets/index.hbs]: Remove it

Comment (snitin315) [reply app/templates/events/view/tickets/index.hbs]: only line 20  or the whole div

Comment (kushthedude) [reply app/templates/events/view/tickets/index.hbs]: the extra line

File: app/templates/events/view/tickets/index.hbs [hunk 2]
```diff
@@ -121,8 +122,8 @@
 </div>
 <br>
 
-<div class="row">
-  <table class="ui stackable celled unstackable table">
+<div class="row x-scrollable">
+  <table class="ui stackable celled unstackable table" >
```
Comment (kushthedude) [inline app/templates/events/view/tickets/index.hbs]: no need for extra space in here

**Comment by snitin315** [comment]
@kushthedude  fixed #3605 in this PR as well. below is the corresponding screenshot.Please review

Screenshot from 2019-11-07 21-23-37

**Comment by mariobehling** [comment]
Thanks a lot for this PR. Please help to get some important issues out of the way in this project. We have labeled these issues here: https://github.com/fossasia/open-event-frontend/issues?q=is%3Aissue+is%3Aopen+label%3A%22Priority%3A+High%22 Our goal is to focus on these issues.

**Pull request merged**